{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py

```diff
@@ -1,8 +1,12 @@
 # Django settings for project project.
-from os import environ
-from django.core.exceptions import ImproperlyConfigured
+from environ import Env, Path
 import pathlib
+from django.core.exceptions import ImproperlyConfigured
 
+env = Env()
```

Comment:
Yeah, at heart it just wraps `os.envrion` and optionally reads in a file. I should have put this a line lower, I meant in the `.read_env` call.